src/Microsoft.IdentityModel.Tokens/X509SecurityKey.cs

```diff
@@ -54,6 +54,7 @@ public X509SecurityKey(X509Certificate2 certificate)
 
             _certificate = certificate;
             KeyId = certificate.Thumbprint;
+            X5t = Base64UrlEncoder.Encode(certificate.Thumbprint);
```

Comment:
I think this should be Base64UrlEnoder.Encode(certificate.GetCertHash()) according to https://tools.ietf.org/html/rfc7517#section-4.8